### DISCUSSION TITLE
Different pins for SPI display as for SPI touch????

### DISCUSSION THREAD

**Discussion by Olivierdemoustier** [Bodmer/TFT_eSPI#2407] [General]
I have bought a very nice Development Board with touch screen and ESP32 , all in one.
But SPI for display use different pins as SPI for touch. How can I configure this in User_Setup.h ?
https://nl.aliexpress.com/item/1005004502250619.html?spm=a2g0o.order_list.order_list_main.46.523079d2QwYifT&gatewayAdapt=glo2nld

**Comment by Bodmer** [comment]
You will need to use an independant library and do not define TOUCH_CS in the setup. Try this one:
https://github.com/achillhasler/TFT_eTouch
